### PR TITLE
Pack & Rule changes

### PR DESCRIPTION
**Pull request by nkulig**
### Background

The SE team went through each pack and associated rules in an effort to remove detections that **require** code changes. This was causing pain for those prospects and/or customers that leverage Packs because simply enabling a Pack could generate a ton of noise (because we require certain changes, example.org -> actual domain, for example). 

### Changes

Certain rules have been removed from a Pack, and the metadata has been changed to enabled: false.

### Testing

We went through and tested each Pack, individually, against an SE dev instance to ensure that the Packs still upload, test and function correctly.

### CODE REVIEW

**Comment by andrea-youwakim** [comment]
some merge conflicts exists! do you mind resolving and re-pushing? 

thanks!

**Review by andrea-youwakim — DISMISSED** [review]
incredibly thorough work! thanks so much for your help on this guys!

**Comment by andrea-youwakim** [comment]
> 

please be sure to fix the merge conflicts and re-request review on those changes, but besides that the changes look really good!
thanks again!

File: packs/panther.yml [hunk 1]
```diff
@@ -7,9 +7,13 @@ PackDefinition:
     - Panther.SAML.Modified
     - Panther.Sensitive.Role
     - Panther.User.Modified
+    - Confluence.0DayIPs
```
Comment (edyesed) [inline packs/panther.yml:10]: Action needed: 
* `Confluence.0DayIPs` -> This is a source IP 0day detection. I advise that we do not include it in the pack, as source IP based detections age rapidly. 
* `IOC.Sunburst` -> these detections are aging similarly, though they are hash or domain based. I would advise that we do not include these either.

File: packs/panther.yml [hunk 2]
```diff
@@ -7,9 +7,13 @@ PackDefinition:
...
+    - IOC.SunburstFQDNIOCs
+    - IOC.SunburstSHA256IOCs
     # Data Model
     - Standard.Panther.Audit
     # Helpers
     - panther_base_helpers
     - panther_event_type_helpers
+    - panther_iocs
```
Comment (edyesed) [inline packs/panther.yml:18]: if we do not include the rapid-response detections, we should also not include this global helper.

File: packs/zendesk.yml [hunk 3]
```diff
@@ -11,9 +11,6 @@ PackDefinition:
     - Zendesk.UserAssumption
     - Zendesk.UserRoleChanged
     - Zendesk.UserSuspension
-    # Standard rules that apply to zendesk
```
Comment (edyesed) [inline packs/zendesk.yml:14]: No action needed:

We opted to include Standard. rules only in the standard pack. This eases dependency resolution packaging. Additionally the standard rules will automagically act on only the enabled logtype for which they are configured to act upon.

File: rules/box_rules/box_item_shared_externally.yml [hunk 4]
```diff
@@ -8,6 +8,7 @@ LogTypes:
 Tags:
   - Box
   - Exfiltration:Exfiltration Over Web Service
+  - Configuration Required
```
Comment (edyesed) [inline rules/box_rules/box_item_shared_externally.yml:11]: No action needed: 
panther_box_helpers require configuration. this detection doesn't directly require config, and the helper it imports does

File: rules/cloudflare_rules/cloudflare_firewall_ddos.yml [hunk 5]
```diff
@@ -7,7 +7,6 @@ LogTypes:
   - Cloudflare.Firewall
 Tags:
   - Cloudflare
```
Comment (edyesed) [inline rules/cloudflare_rules/cloudflare_firewall_ddos.yml:9]: yep, this detection does not require configuration

File: rules/cloudflare_rules/cloudflare_firewall_high_volume_events_blocked.yml [hunk 6]
```diff
@@ -2,12 +2,11 @@ AnalysisType: rule
 Filename: cloudflare_firewall_high_volume_events_blocked.py
 RuleID: Cloudflare.Firewall.HighVolumeEventsBlocked
 DisplayName: Cloudflare - High Volume Events Blocked
-Enabled: false
+Enabled: true
```
Comment (edyesed) [inline rules/cloudflare_rules/cloudflare_firewall_high_volume_events_blocked.yml:5]: action needed: 

This detection purports to detect high volume events, though the trigger for it is that the cloudflare firewall action is `block`.  
* We should confirm that the only reason for `block` is high-volume events.  Without that confirmation, I suspect we should not default this to `enabled: true` due to volume of alerts
* We should also tune the dedup() function for this. With the current title, any block event globally across all properties is going to suppress additional notifications. If multiple web properties were targeted at once, we'd group alerts via dedup, which might make the scale of the attack less clear.

File: rules/cloudflare_rules/cloudflare_firewall_high_volume_events_blocked_greynoise.yml [hunk 7]
```diff
@@ -2,13 +2,12 @@ AnalysisType: rule
 Filename: cloudflare_firewall_high_volume_events_blocked_greynoise.py
 RuleID: Cloudflare.Firewall.HighVolumeEventsBlockedGreyNoise
 DisplayName: Cloudflare High Volume Events Blocked - GreyNoise
-Enabled: false
+Enabled: true
```
Comment (edyesed) [inline rules/cloudflare_rules/cloudflare_firewall_high_volume_events_blocked_greynoise.yml:5]: GreyNoise basic is enabled by default.

File: rules/cloudflare_rules/cloudflare_httpreq_bot_high_volume.yml [hunk 8]
```diff
@@ -7,7 +7,6 @@ LogTypes:
   - Cloudflare.HttpRequest
 Tags:
   - Cloudflare
```
Comment (edyesed) [inline rules/cloudflare_rules/cloudflare_httpreq_bot_high_volume.yml:9]: Action needed, I think: 

While this detection doesn't require configuration, there is a logical bug in it that a `BotScore` of 0 means "not computed". We should be returning False when BotScore is zero or  greater than 30 https://developers.cloudflare.com/bots/concepts/bot-score/#bot-groupings

Additionally, I think we should add a conditional for the action being block.

File: rules/cloudflare_rules/cloudflare_httpreq_bot_high_volume_greynoise.yml [hunk 9]
```diff
@@ -2,13 +2,12 @@ AnalysisType: rule
 Filename: cloudflare_httpreq_bot_high_volume_greynoise.py
 RuleID: Cloudflare.HttpRequest.BotHighVolumeGreyNoise
 DisplayName: Cloudflare Bot High Volume GreyNoise
-Enabled: false
+Enabled: true
```
Comment (edyesed) [inline rules/cloudflare_rules/cloudflare_httpreq_bot_high_volume_greynoise.yml:5]: action needed: 
this has the same logical bug as the other httpreq_bot_high_volume.. also, there is substantial overlap between both detections and their greynoise variants. could we consolidate into one detection( presumably the greynoise version )

File: rules/indicator_creation_rules/new_aws_account_logging.yml [hunk 10]
```diff
@@ -2,7 +2,7 @@ AnalysisType: rule
 Filename: new_aws_account_logging.py
 RuleID: Standard.NewAWSAccountCreated
 DisplayName: New AWS Account Created
-Enabled: false
+Enabled: true
```
Comment (edyesed) [inline rules/indicator_creation_rules/new_aws_account_logging.yml:5]: this is fine. defaults to three days to consider an account "new"

File: rules/indicator_creation_rules/new_user_account_logging.yml [hunk 11]
```diff
@@ -8,7 +8,7 @@ AnalysisType: rule
 Filename: new_user_account_logging.py
 RuleID: Standard.NewUserAccountCreated
 DisplayName: New User Account Created
-Enabled: false
+Enabled: true
```
Comment (edyesed) [inline rules/indicator_creation_rules/new_user_account_logging.yml:11]: also ok

File: rules/standard_rules/malicious_sso_dns_lookup.yml [hunk 12]
```diff
@@ -11,6 +11,8 @@ LogTypes:
   - Zeek.DNS
 Severity: Medium
 Threshold: 1000 # dedup & threshold is high to prevent alert storms from FPs
+Tags:
```
Comment (edyesed) [inline rules/standard_rules/malicious_sso_dns_lookup.yml:14]: 👍

**Review by edyesed — CHANGES_REQUESTED** [review]
I had some questions, particularly on the CloudFlare detections ( though the things I find curious there are not part of this PR ). I left comments on all items, if they say `no action needed` then it was record keeping for my review.  

Let me know your thoughts on the other items. thanks! 👍

File: packs/gcp_audit.yml [hunk 13]
```diff
@@ -7,6 +7,14 @@ PackDefinition:
     - GCP.GCS.Public
     - GCP.IAM.CorporateEmail
     - GCP.IAM.CustomRoleChanges
+    - GCP.IAM.OrgFolderIAMChanges
     - GCP.SQL.ConfigChanges
-    - GCP.UnusedRegions
+    - GCP.K8s.ExecIntoPod
```
[the remark this answers is not in the document]
Comment (jzandona) [reply packs/gcp_audit.yml:12]: Will update `GCP.K8s.ExecIntoPod`, good catch.

File: packs/onepassword.yml [hunk 14]
```diff
@@ -5,14 +5,8 @@ DisplayName: Panther OnePassword Pack
...
     - Standard.OnePassword.SignInAttempt
-    # Standard Detections Supporting 1Password
-    - Standard.BruteForceByIP
     # 1Password Specific Rules
-    - OnePassword.Sensitive.Item
-    - OnePassword.Lut.Sensitive.Item
     - OnePassword.Unusual.Client
     # Supporting Global Helpers
     - panther_base_helpers
-    - panther_event_type_helpers
```
[the remark this answers is not in the document]
Comment (nkulig) [reply packs/onepassword.yml:18]: Nice catch. Adding it back!

File: packs/panther.yml (hunk 1 above)
Comment (nkulig) [reply packs/panther.yml:10]: Noted. Will remove!

File: rules/cloudflare_rules/cloudflare_firewall_high_volume_events_blocked.yml (hunk 6 above)
Comment (nkulig) [reply rules/cloudflare_rules/cloudflare_firewall_high_volume_events_blocked.yml:5]: Flipping this to `false` for now while we update this detection in another branch.

File: rules/cloudflare_rules/cloudflare_httpreq_bot_high_volume.yml (hunk 8 above)
Comment (nkulig) [reply rules/cloudflare_rules/cloudflare_httpreq_bot_high_volume.yml:9]: Flipping this to false for now while we update this detection in another branch.

File: rules/cloudflare_rules/cloudflare_httpreq_bot_high_volume_greynoise.yml (hunk 9 above)
Comment (nkulig) [reply rules/cloudflare_rules/cloudflare_httpreq_bot_high_volume_greynoise.yml:5]: Flipping this to false for now while we update this detection in another branch.

**Comment by nkulig** [comment]
Seriously appreciate your thoughts, comments, and assistance here @edyesed and @andrea-youwakim

**Review by edyesed — APPROVED** [review]
major update of pack content, changes are too many to list in this comment.

**Review by andrea-youwakim — APPROVED** [review]
fantastic packs cleanup work done by this team, you guys rock, thanks for your help!

**Pull request merged**